spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -421,6 +421,26 @@ def count
       end.to raise_error(/#let or #subject called without a block/)
     end
 
+    it 'emits a warning when declared multiple times' do
+      line = nil
+      expect {
+        RSpec.describe do
+          let(:foo) { 'first value' }
+          let(:foo) { 'second value' }; line = __LINE__
+        end
+      }.to output(a_string_including("#foo", "let(:foo)", "definition at #{__FILE__}:#{line}")).to_stderr
```

Comment:
So I realized something about our use of `line` here...when the `a_string_including` matcher is created, `line` is `nil` because the `expect` block hasn't executed yet.  So the last arg to `a_string_including` is the same as `"definition at #{__FILE__}:#{nil}"` which passes because `nil.to_s == ""`.  I think we're better off using `#{__LINE__ - 2}` instead of a `line` local to avoid this issue.